tests/integration/createPackage/createPackage.spec.ts

```diff
@@ -60,6 +65,95 @@ describe('tiles', function () {
       expect(createJobSpy).toHaveBeenCalledTimes(1);
       expect(resposne.status).toBe(httpStatusCodes.OK);
     });
+
+    it(`should return 200 status code and the exists un-cleaned completed job's callback (with original bbox of request)`, async function () {
+      checkForDuplicateSpy.mockRestore();
+
+      const expirationTime = new Date();
+      const body: ICreatePackage = {
+        dbId: layerFromCatalog.id,
+        bbox: [34.811938017107494, 31.95475033759175, 34.82237261707599, 31.96426962177354],
+        targetResolution: 0.0000429153442382812,
+        callbackURLs: ['http://example.getmap.com/callback'],
+        crs: 'EPSG:4326',
+        priority: 0,
+      };
+      const origCallback = {
+        dbId: layerFromCatalog.id,
+        fileUri: 'http://example.getmap.com/callback',
+        success: true,
+        fileSize: 10,
+        requestId: 'string',
+        errorReason: '',
+        packageName: 'string',
+        expirationTime: expirationTime,
+        targetResolution: 0.123,
+        status: OperationStatus.COMPLETED,
+        bbox: [-180, -90, 90, 180],
+      };
+
+      const expectedCompletedCallback = {
+        dbId: layerFromCatalog.id,
+        fileUri: 'http://example.getmap.com/callback',
+        success: true,
+        fileSize: 10,
+        requestId: 'string',
+        errorReason: '',
+        packageName: 'string',
+        expirationTime: expirationTime,
+        targetResolution: 0.123,
```

Comment:
body.targetResolution is the expected res (should not affect the src code)

Reply:
its part of the static mock data, doesn't affect the src and logic, the test validating the logic of providing the right bbox (of the request)